vendor/github.com/containers/image/v5/docker/docker_transport.go

```diff
@@ -113,6 +137,9 @@ func (ref dockerReference) DockerReference() reference.Named {
 // not required/guaranteed that it will be a valid input to Transport().ParseReference().
 // Returns "" if configuration identities for these references are not supported.
 func (ref dockerReference) PolicyConfigurationIdentity() string {
+	if ref.isUnknownDigest {
+		return ref.ref.Name()
```

Comment:
ACK.

`PolicyConfigurationNamespaces` should, for `isUnknownDigest`, probably verify that the first item is equal to `ref.ref.Name()` and remove it, so that we don’t process the same policy scope twice.